calico-cloud_versioned_docs/version-18/release-notes/index.mdx

```diff
@@ -9,8 +9,18 @@ title: Release notes
 
 ### New features and enhancements
 
+#### Support for RKE2
+
+This release comes with support for installing Calico Cloud on RKE2 clusters.
+
+For more information, see the [requirements for RKE2](../get-started/connect/requirements/rke2.mdx)
+
 ### Known issues
 
+* For clusters running on RKE2, having the Image Assurance runtime view enabled may cause Calico Cloud upgrades to fail.
```

Comment:
Would reword to say `will cause a connect to Calico Cloud to fail if Image Assurance Runtime View was enabled prior.`

Reply:
Reworded.